ReportingServicesTools/Libraries/library.ps1

```diff
@@ -111,10 +111,20 @@ namespace Microsoft.ReportingServicesTools
         /// </summary>
         SQLServer2017 = 14,
 
+        /// <summary>
+        /// SQL Server 2019
+        /// </summary>
+        SQLServer2019 = 15,
+
+        /// <summary>
+        /// SQL Server 2022
+        /// </summary>
+        SQLServer2022 = 16,
+
         /// <summary>
         /// SQL Server vNext
         /// </summary>
-        SQLServervNext = 15
+        SQLServervNext = 17
```

Comment:
Remove SQLServervNext, we added that when SQL 2019 was announced but without a release date or branding.

Reply:
Removed. Note the build is still failing with an error that looks intermittent on your builds? I see previous builds failing with the same issue that it looks like someone just kept hitting the retry button on until it worked... hopefully that's all my failure is?

Reply:
I’ll investigate the build issues as well.